render-app/src/main/java/org/janelia/alignment/spec/TileSpec.java

```diff
@@ -118,6 +118,10 @@ public Double getZ() {
         return z;
     }
 
+    public Integer getIntegralZ() {
```

Comment:
I would probably name this `getIntegerZ()` to remove ambiguities with the mathematical operation of integration.

Reply:
good point - done